### PR TITLE
Lodestar finance strategies [lodestar-staked-lp]

### PR DESCRIPTION
**Pull request by 0xAppo**
Fixes # .

Changes proposed in this pull request:
- Add staked LP strategy

### CODE REVIEW

File: src/strategies/lodestar-staked-lp/index.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      (parseFloat(formatUnits(lpTokenReserves._reserve1, tokenDecimals)) /
+        parseFloat(formatUnits(lpTokenTotalSupply, lpTokenDecimals))) *
+      2;
+  } else {
+    tokenWeight = 0;
+  }
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+
+  options.stakingPoolAddresses.forEach(stakingPoolAddress => {
```
Comment (ChaituVR) [inline src/strategies/lodestar-staked-lp/index.ts:70]: Hi @0xAppo Everything looks good, Can throw some error here if someone passes too many addresses? Thanks

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/lodestar-staked-lp/index.ts (hunk 1 above)
Comment (0xAppo) [reply src/strategies/lodestar-staked-lp/index.ts:70]: Hi there, would this method be acceptable?

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Need same changes as in here https://github.com/snapshot-labs/snapshot-strategies/pull/1030

**Comment by 0xAppo** [comment]
I think I fixed it, let me know if that works

File: src/strategies/lodestar-staked-lp/index.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    if (options.stakingPoolAddresses.length >= 10) {
+      throw new Error('Too many stake pool addresses provided.')
+    }
```
Comment (ChaituVR) [inline src/strategies/lodestar-staked-lp/index.ts:73]: Can move this to before forEach?

Comment (0xAppo) [reply src/strategies/lodestar-staked-lp/index.ts]: Done.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**